frontend/src/data/utils/datasetutils.ts

```diff
@@ -144,10 +144,39 @@ export function percent(numerator: number, denominator: number): number | null {
 
 /** Finds expected value of an ailment based on a population sample. */
 export function estimateTotal(
+  sRow: Row,
+  acsState: IDataFrame,
+  stateTotals: IDataFrame,
   sample_count: number,
   sample_population: number,
   total_population: number
 ): number {
+  if (
+    sRow.race_and_ethnicity === UNKNOWN_RACE ||
+    sRow.race_and_ethnicity === UNKNOWN
+  ) {
+    const state_population = acsState
+      .where((row) => row.fips === sRow.fips)
+      .where((row) => row.race_and_ethnicity === ALL)
+      .resetIndex()
+      .at(0);
+
+    if (state_population !== undefined) {
+      total_population = state_population.population;
+    }
+
+    const sample_population = stateTotals
+      .where((row) => row.fips === sRow.fips)
+      .resetIndex()
+      .at(0).total_sample_size;
+
+    return sample_count == null ||
+      sample_population == null ||
```

Comment:
I *think* this can be simplified by using bool coalescence.

 !!sample_population ? Math.round(((sample_count ?? 0)/ sample_population) * total_population) : 0